x-pack/plugins/aiops/public/components/log_rate_analysis/item_filter_popover.tsx

```diff
@@ -27,26 +27,34 @@ import {
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
 
-import { FieldFilterApplyButton } from './field_filter_apply_button';
+import { ItemFilterApplyButton } from './item_filter_apply_button';
 
-interface FieldFilterPopoverProps {
+interface ItemFilterPopoverProps {
   disabled?: boolean;
   disabledApplyButton?: boolean;
-  uniqueFieldNames: string[];
-  onChange: (skippedFields: string[]) => void;
+  disabledApplyTooltipContent?: string;
+  helpText: string;
+  itemType?: string;
```

Comment:
itemType can be "columns" | "fields" here

Reply:
Removed the need for itemType completely by passing in the necessary labels. Updated in 72f120e9a39c579999d4b3aabcf548ff88d6fdf0